spec/savon/soap_fault_spec.rb

```diff
@@ -49,6 +49,11 @@
     it "returns false unless the HTTP response contains a SOAP fault" do
       expect(Savon::SOAPFault.present? new_response).to be_falsey
     end
+
+    it "returns true if the HTTP response contains a SOAP fault with invalid ecoded char" do
```

Comment:
```suggestion
    it "returns true even if the HTTP response contains a SOAP fault with invalid ecoded char" do
```

Reply:
I added an `even` in order to underline that this is an additional condition that the code's able to handle.